src/forms/ShowOnYes.js

```diff
@@ -27,7 +27,6 @@ class ShowOnYes extends React.Component {
   }
 
   handleChange = (evt, inputProps) => {
-
     if (inputProps.value === 'Yes') {
```

Comment:
Instead of changing `value` in `ControlledRadioYesNo` to `checked`, could we keep the `handleChange` that you deleted (maybe changed to an arrow function) and change this line to check whether `inputProps.value` is true? Then I think we wouldn't need the special case in `convertForUpdate` to handle `checked`.

(I guess either way it's kind of a special case, but at least this way we'd only have to handle converting 'Yes' or 'No' to a boolean in one place.)

Reply:
I think what you're saying is to change `ControlledRadioYesNo` so that it converts the value of `value` instead of doing that in `ShowOnYes`. I thought I had tried that and run into a problem, but I'm having some trouble remembering the details right now.

Given that, I'll go ahead and try it and see what happens!

Reply:
Also, I may have thought that keeping `Radio`s behaving in the expected manner all the way through, even though I don't like it, was the way to go. Who can know what yesterday-knod was thinking!? No one, that's who.